egs/wsj/s5/steps/nnet3/components.py

```diff
@@ -461,5 +461,72 @@ def AddClstmLayer(config_lines,
             'dimension':output_dim
             }
 
+# Equations that specify a GRU layer:
```

Comment:
Expand the acronym one time in every script. It would be easier for people to follow if you refer the paper which proposed this unit.